src/Library/demos/Label/main.py

```diff
@@ -0,0 +1,61 @@
+import workbench
+
+label = workbench.builder.get_object("label")
+justification_row = workbench.builder.get_object("justification_row")
+style_row = workbench.builder.get_object("style_row")
+single_line_switch = workbench.builder.get_object("single_line_switch")
+
+style_classes = [
+    "none",
+    "title-1",
+    "title-2",
+    "title-3",
+    "title-4",
+    "monospace",
+    "accent",
+    "success",
+    "warning",
+    "error",
+    "heading",
+    "body",
+    "caption-heading",
+    "caption",
+]
+
+short_label = "<b>Lorem ipsum</b> dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore disputandum putant."
+
+long_label = """     <b>Lorem ipsum</b> dolor sit amet, consectetur adipiscing elit,
+sed do eiusmod tempor incididunt ut labore et dolore magnam aliquam quaerat voluptatem.
+Ut enim mortis metu omnis quietae vitae status perturbatur,
+et ut succumbere doloribus eosque humili animo inbecilloque ferre miserum est,
+ob eamque debilitatem animi multi parentes, multi amicos, non nulli patriam,
+plerique autem se ipsos penitus perdiderunt, sic robustus animus et excelsus omni."""
+
+label.set_label(short_label)
+
+single_line_switch.connect(
+    "notify::active",
+    lambda *_: label.set_label(
```

Comment:
I think this is a bit too complicated for a lambda, please use a function and proper if-else.